### ISSUE TITLE
Issue with sending attachments

### ISSUE THREAD

**Issue by RamonSmit** [SparkPost/php-sparkpost#134]
Hi there, 

I've included the new version of the Sparkpost PHP library to my project, but unfortunately I can't send attachments. I get this error: **"field 'content.attachments' is of type 'json_object', but needs to be of type 'json_array'"**

And I don't really know what I'm doing wrong in here.

Here is te code with the attachment: 
```
$promise = $sparky->transmissions->post([
    'content' => [
        'from' => [
            'name' => 'Ramon Smit',
            'email' => 'ramon@daltcore.com',
        ],
        'subject' => 'DALTCORE: Uw opgevraagde SEPA mandaat',
        'html' => (string) view('email.user.mandate')->set(compact('user'))->render(),
		'attachments' => [
			"type" => "application/pdf",
			"name" => "mandaat.pdf",
			"data" => base64_encode($dompdf->output()),
		],
    ],
    'recipients' => [
        [
            'address' => [
                'name' => $user->firstname . ' ' . $user->lastname,
                'email' => $user->email,
            ],
        ],
    ],
]);
```

How do I solve this problem? 

Cheers,
Ramon Smit

**Comment by ewandennis** [comment]
@RamonSmit - looks like your attachments field might be a single attachment object `[ "type"=> ... ]` instead of an array of attachments objects: `[ [ "type" => ...], ... ]`.

**Comment by RamonSmit** [comment]
@ewandennis - Your answer is just pure magic! Been fighting this problem for 2...

Well, You just solved #134 I would say!
